### PR TITLE
Add some missing docs

### PR DESCRIPTION
**Pull request by jerith**
I'm going through the output of `make coverage` and adding a bunch of things that it complains about.

### CODE REVIEW

File: seaworthy/pytest/fixtures.py [hunk 1]
```diff
@@ -35,9 +35,7 @@ def fixture():
     return fixture
 
 
-"""
-Default fixture for the ``DockerHelper``. Has module scope.
-"""
+#: Default fixture for the :class:`.DockerHelper`. Has module scope.
```
Comment (JayH5) [inline seaworthy/pytest/fixtures.py]: `#:` 🤔 learn something new every day.

File: seaworthy/client.py [hunk 2]
```diff
@@ -12,6 +12,14 @@ def _path_segments(url, path_str):
 
 
 class ContainerHttpClient:
+    """
+    HTTP client for a specific container.
+
+    In most cases, these should be obtained from
+    :meth:`.ContainerDefinition.http_client` instead of
```
Comment (JayH5) [inline seaworthy/client.py]: this line seems a bit short?

File: seaworthy/checks.py [hunk 3]
```diff
@@ -13,12 +13,21 @@
 
 @contextmanager
 def docker_client():
+    """
+    A context manager that creates and cleans up a docker API client.
```
Comment (JayH5) [inline seaworthy/checks.py]: Tiny nitpick: can we try say `Docker` consistently?

File: seaworthy/pytest/fixtures.py (hunk 1 above)
Comment (jerith) [reply seaworthy/pytest/fixtures.py]: You can also write the docstring *after* the attribute being documented, but I find that more confusing.

File: seaworthy/ps.py [hunk 4]
```diff
@@ -4,18 +4,27 @@
...
-    pass
+    """
+    Exception indicating problems operating on process lists and trees.
+    """
 
 
 @attr.s
 class PsRow(object):
+    """
+    Representation of a process list entry.
```
Comment (JayH5) [inline seaworthy/ps.py]: Nitpick: this is technically accurate but possibly not that easy to understand. Could maybe just add a sentence like `Contains details about a single running process.`

**Pull request merged**